src/acceptance/java/io/codurance/pouch/DatabaseHelper.java

```diff
@@ -57,8 +59,8 @@ void clearResources() {
 
     void insertResource(Resource resource) {
         try (PreparedStatement preparedStatement = connection.prepareStatement("INSERT INTO RESOURCE (ID, ADDED, TITLE, URL) VALUES (?, ?, ?, ?)")) {
```

Comment:
My instincts tell me to extract the SQL statement to a variable.

Reply:
Yes, that could improve clarity, I agree.